lib/services/log.js

```diff
@@ -1,45 +1,23 @@
 'use strict';
 
-var _ = require('lodash'),
-  chalk = require('chalk'),
-  path = require('path'),
-  winston = require('winston'),
-  util = require('util'),
-  logLevel = process.env.LOG || 'info';
+const clayLog = require('clay-log'),
+  pkg = require('../../package.json');
+var amphoraSearchLogInstance;
 
-// default to console logger, but pretty-print it
-winston.remove(winston.transports.Console);
-winston.add(winston.transports.Console, {
-  colorize: true
+// Initialize the logger
+clayLog.init({
```

Comment:
will this always run before the `clayLog.setup()` functions are run, in various other files? Or does it matter? (this isn't in the docs anywhere)

Reply:
This is run once and is just the setup. Since it's globally scoped it is run the first time the file is required and then maintained in memory. I could wrap this in a function that setup could check has been called.

I don't _think_ this needs to be in the docs for this repo anywhere because this is an internal module only and not exposed to external modules.